src/qililab/calibration/calibration_controller.py

```diff
@@ -12,7 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# pylint: disable=anomalous-backslash-in-string
```

Comment:
Docstring no la toque illo! We have to use the ones from the main autocalibration branch, after checking this works in the lab 👍